### PR TITLE
Consistent blog card on website

### LINKED ISSUE #1299: UI - Blog card uneven size
The blog card in https://dicedb.io/ is of uneven size. The button "Read post" feels off due the same reason.

image

### PR DESCRIPTION
**Pull request by realChakrawarti**
Fixes #1299 

| Before | After |
| - | - |
| image | image |

### CODE REVIEW

File: docs/public/logo.png
Comment (lucifercr07) [inline docs/public/logo.png:1]: Any reasons for updating this file?

**Review by lucifercr07 — CHANGES_REQUESTED** [review]

File: docs/package-lock.json
[the remark this answers is not in the document]
Comment (realChakrawarti) [reply docs/package-lock.json:1]: Refer: https://docs.npmjs.com/cli/v10/configuring-npm/package-lock-json

File: docs/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "sharp": "^0.32.5"
   },
   "devDependencies": {
+    "astro": "^4.16.13",
```
[the remark this answers is not in the document]
Comment (realChakrawarti) [reply docs/package.json:23]: Here is the reason why I added the `astro` as a dev dependency and is indeed required:

TL;DR: I was not able to run the project without adding `astro` as a dependency.

<hr />

If you check the scripts in `package.json` [npm run dev](https://github.com/DiceDB/dice/blob/114bc49c938728043ee43bed91e66fbe43c2f07f/docs/package.json#L6) for example, it depends on `astro` to compile and similarly to build the project. Here are the ways to do that without adding it:

- One could install `astro` manually & globally, [refer](https://docs.npmjs.com/downloading-and-installing-packages-globally) which will let them compile, build, preview without adding it as a dependency in the project but this isn't documented anywhere.
- I also don't see a postinstall script which installs astro globally.  

I referred to astro official repo: https://github.com/withastro/astro which states how to do so manually. And I believe, only the static built files are served by netlify which means `astro` isn't needed during the runtime and it makes sense to be a dev dependency.

I hope it clear why I added it and is required.

If you have any further insight please do share. Thanks for reviewing.

**Review by lucifercr07 — COMMENTED** [review]
@realChakrawarti thanks for contributing, please resolve the conflicts. LGTM.

**Review by lucifercr07 — APPROVED** [review]
